composer.json

```diff
@@ -98,13 +96,18 @@
         "mezzio": "laminas --ansi",
         "check": [
             "@cs-check",
-            "@test",
-            "@analyze"
+            "@test"
         ],
-        "analyze": "phpstan analyze -l 3 -c ./phpstan.installer.neon ./src ./config",
         "clear-config-cache": "php bin/clear-config-cache.php",
-        "cs-check": "phpcs -s",
-        "cs-fix": "phpcbf",
+        "enable-codestandard": "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run",
+        "cs-check": [
+            "@enable-codestandard",
+            "phpcs -s"
+        ],
+        "cs-fix": [
+            "@enable-codestandard",
+            "phpcbf"
+        ],
```

Comment:
This was proposed by @boesing in #57, and I expanded on it here as we need to install without plugins or scripts executing. These changes allow running phpcs/phpcbf even whn plugins are not installed.

Reply:
TBH, we could automate some of this in the laminas CI setup (to auto-push to contributor branches after message confirmation, potentially)

Reply:
This is only necessary on projects where we are using the `--disable-plugins` and/or `--disable-scripts` switches. We _might_ want to detect that, but I think there's only going to be a very small number of places where it will be useful; since there are ways to work around it with the existing tooling, I'm not sure if it makes sense.

Reply:
I wonder if this is somehow solvable in another way.
I remember that phpstan has some similar magic and most of my colleagues which do use phpstan in other projects (at work) got rid of that composer plugin magic and added the plugins path to their config.

Yes, I know that would mean we would have to change all repositories if we wanted to roll that out for all components but IMHO its only necessary here due to the fact that we do, in fact, ignore composer plugins intentional.

So instead of still relying on composer, we could simply change the path to the coding standard in the codestyle config instead. WDYT?